boards/alientek-pandora/include/board.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2021 Luo Jia (HUST IoT Security Lab)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_alientek-pandora ALIENTEK-PANDORA
```

Comment:
You want to move this to `doc.txt` where you can also provide a link to the pinout of the board.

Reply:
Yes, it should be moved to doc.txt